tests/mentorship_relation/test_api_list_relations.py

```diff
@@ -0,0 +1,118 @@
+    # User 2 is the receiver
+    def setUp(self):
+        super(TestListMentorshipRelationsApi, self).setUp()
+
+        self.first_user = UserModel(
+            name=user1['name'],
+            email=user1['email'],
+            username=user1['username'],
+            password=user1['password'],
+            terms_and_conditions_checked=user1['terms_and_conditions_checked']
+        )
+        self.second_user = UserModel(
+            name=user2['name'],
+            email=user2['email'],
+            username=user2['username'],
+            password=user2['password'],
+            terms_and_conditions_checked=user2['terms_and_conditions_checked']
+        )
+
+        # making sure both are available to be mentor or mentee
+        self.first_user.need_mentoring = True
+        self.first_user.available_to_mentor = True
+        self.second_user.need_mentoring = True
+        self.second_user.available_to_mentor = True
+
+        self.notes_example = 'description of a good mentorship relation'
+
+        self.now_datetime = datetime.now()
+        self.past_end_date_example = self.now_datetime - timedelta(weeks=5)
+        self.future_end_date_example = self.now_datetime + timedelta(weeks=5)
+
+        db.session.add(self.first_user)
+        db.session.add(self.second_user)
+        db.session.commit()
+
+        # create new mentorship relation
+
+        self.past_mentorship_relation = MentorshipRelationModel(
+            action_user_id=self.first_user.id,
+            mentor_user=self.first_user,
+            mentee_user=self.second_user,
+            creation_date=self.now_datetime.timestamp(),
+            end_date=self.past_end_date_example.timestamp(),
+            state=MentorshipRelationState.PENDING,
+            notes=self.notes_example
+        )
+
+        self.future_pending_mentorship_relation = MentorshipRelationModel(
+            action_user_id=self.first_user.id,
+            mentor_user=self.first_user,
+            mentee_user=self.second_user,
+            creation_date=self.now_datetime.timestamp(),
+            end_date=self.future_end_date_example.timestamp(),
+            state=MentorshipRelationState.PENDING,
+            notes=self.notes_example
+        )
+
+        self.future_accepted_mentorship_relation = MentorshipRelationModel(
+            action_user_id=self.first_user.id,
+            mentor_user=self.first_user,
+            mentee_user=self.second_user,
+            creation_date=self.now_datetime.timestamp(),
+            end_date=self.future_end_date_example.timestamp(),
+            state=MentorshipRelationState.ACCEPTED,
+            notes=self.notes_example
+        )
+
+        db.session.add(self.past_mentorship_relation)
+        db.session.add(self.future_pending_mentorship_relation)
+        db.session.add(self.future_accepted_mentorship_relation)
+        db.session.commit()
+
+    def test_list_previous_mentorship_relations(self):
```

Comment:
could you please change the 'previous' to 'past' to keep the consistency?

Reply:
Ok so should I change `previous` -> `past` in the app as well or just in the test?

Reply:
just use one term everywhere to keep consistency. For now it's ok to change places relevant to this PR, but is there are many places, it is better to send a different PR :) either "previous" or "past". Note that you are using "previous" in the request.

Reply:
yes